README.md

```diff
@@ -99,123 +41,82 @@ After you're finished please remove all the comments and instructions!
 
 > Describe between 1-3 key features of the application.
```

Comment:
- [ ] Please remove all these placeholder texts from your Readme , they are there to help you fill in the details and should not be present on your committed or professional readme version :+1: